homeassistant/components/solarlog/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "title": "Solar-Log",
+    "step": {
+      "user": {
+        "title": "Define your Solar-Log connection",
+        "data": {
+          "host": "The hostname or ip-address of your Solar-Log device (without http://)",
+          "name": "The prefix to be used for your Solar-Log sensors"
```

Comment:
Can we query this from the Solar-Log installation and thus omit from the form?

Reply:
Just checking if I understand you right. Do you mean that you want to remove this from the (config_flow) form, so only ask for the host? Is it ok to keep the configuration option for ‘name’ in the configuration.yaml?